scripts/stylable-metadata-plugin.js

```diff
@@ -0,0 +1,175 @@
+const path = require('path')
+const fs = require('fs')
+const pathFinder = require('react-autodocs-utils/src/path-finder');
+const parse = require('react-autodocs-utils/src/parser/parse');
```

Comment:
i never really intended these files to be publicly available, not sure it's a good idea to use them directly (talking about `parse` and `visit`).

since you're using `@babel/types` you can also do `@babel/traverse` (for `visit`) & `@babel/parse` (for, well, `parse`)